Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -184,11 +185,15 @@ final class SettingsTableViewController: UITableViewController, IdentifiableClas
                 switchCell.switch?.addTarget(self, action: #selector(dosingEnabledChanged(_:)), for: .valueChanged)
 
                 return switchCell
-            case .notifications:
+            case .alertPermissions:
                 let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
-                cell.textLabel?.text = NSLocalizedString("Alert Permissions", comment: "Title text for Notification & Critical Alert Permissions button cell")
-                if showNotificationsWarning {
-                    cell.detailTextLabel?.text = NSLocalizedString("⚠️", comment: "Warning symbol")
+                if temporaryNewSettingsViewTesting {
```

Comment:
See above comment. This is temporary while this feature is under development.

Reply:
I think it might be helpful, and not a problem, if we just enable this while it's in development as a new row in settings called something like "New Settings (under development)", so design or testers could give feedback on the UI/functionality that does exist earlier.  Also, I changed `temporaryNewSettingsViewTesting ` and `showNotificationsWarning` flags to true, but I still do not see it show up.  What else do I need to do?

Reply:
@ps2 You need to just change `temporaryNewSettingsViewTesting` to true, and then decline either Critical Alerts Permissions or Notifications Permissions.  That said, I'll move it to a temporary button instead of hiding it the way I did.